src/array_approx.rs

```diff
@@ -193,6 +193,3 @@ macro_rules! impl_approx_traits {
 
 #[cfg(feature = "approx")]
 impl_approx_traits!(approx, "**Requires crate feature `\"approx\"`.**");
```

Comment:
I guess you want to keep the macro around for the time when approx 0.6 is relevant?

From a code maintenance perspective, getting the code out of the macro would be preferable IMHO.

Reply:
I'm just adapting necessary changes with as low overhead as possible, unfortunately. I'm going for less than perfect for this release :slightly_smiling_face: 

Keeping the macro doesn't sound so useful no, approx doesn't look like it's going to have a release soon. If you want to fix this, it would be welcome too.